multihost_test/Sanity/script/pam_pwquality.sh

```diff
@@ -0,0 +1,55 @@
+# $1 - user
```

Comment:
I thought the objective was to port all tests to python. Is my assumption incorrect?

Reply:
This is part where we test user password , directly running these scripts on client machine . In many cases its not possible to directly run some commands with our test library. So this part is easiest and safest way to test .

Reply:
This is one of the example where we take same approach :  https://bugzilla.redhat.com/show_bug.cgi?id=1859252

As its not possible to automate with python , its must that we run some script directly on client machine

Reply:
Understood. Thank you.